src/transaction/log_page_buffer.c

```diff
@@ -8350,6 +8350,8 @@ logpb_check_stop_at_time (FILEIO_BACKUP_SESSION * session, time_t stop_at, time_
   return NO_ERROR;
 }
 
+using namespace std;
+
```

Comment:
```suggestion
```
Is it needed?

Reply:
No, it is not removed in the develop PR. It would be nice handle in the other PR cause the appveyor consume too much time. I will handle it soon.

Reply:
I don't think I fully understand what you mean.
> No, it is not removed in the develop PR

I see that it's also not removed in the develop branch. we missed it, unfortunately, right? I think we can remove this while doing backports for other branches, and deal with the develop branch in the following PR.

 >  It would be nice handle in the other PR cause the appveyor consume too much time. I will handle it soon.

I don't understand this.